apptests/appscenarios/suite_test.go

```diff
@@ -39,8 +58,8 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	err := env.Destroy(ctx)
-	Expect(err).ToNot(HaveOccurred())
+	//err := env.Destroy(ctx)
```

Comment:
do we want to comment this out or its only for local testing?

Reply:
For local testsing... Good catch, thanks!